Std/Data/String/Lemmas.lean

```diff
@@ -23,6 +23,16 @@ theorem ext_iff {s₁ s₂ : String} : s₁ = s₂ ↔ s₁.data = s₂.data :=
 
 @[simp] theorem mk_length (s : List Char) : (String.mk s).length = s.length := rfl
 
+@[simp] theorem length_toString (c : Char) : c.toString.length = 1 := rfl
```

Comment:
I think this would make more sense in the `Char` namespace, otherwise the name is too general